src/java.base/share/classes/java/io/SequenceInputStream.java

```diff
@@ -39,7 +40,7 @@
  * and so on, until end of file is reached
  * on the last of the contained input streams.
  *
- * @author  Author van Hoff
+ * @author  Arthur van Hoff
```

Comment:
Good catch. Looking at some other files in the JDK, your change here looks correct.